docs/extensions/tutorials/_posts/1970-01-07-ModifyingRootApp.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/modifying-root-app
+title: Modifying Root App
+section: Tutorials
+---
+
+# Modifying Root App
+
+When making an extension with native dependencies, it's often necessary to change properties outside of the extension. One way we address this need using anchors, which can be used to inject code into files which often need to be modified in order to set up a native dependency.
+
+Another tool we provide is the merging of the extension's app segmenet into the root app. For example, instead of directly modifying the root Android `build.gradle`, you can simply create a `build.gradle` in  the `{{ site.example.devName }}.extension-name/app/android` directory which is then merged into the root `build.gradle`. You can see an example in `shoutem.places`, [here](https://github.com/shoutem/extensions/tree/master/shoutem.places/app/android).
```

Comment:
`build.gradle` doesn't get merged, it defines a new Android module, like a native part of your extension. What does get merged is parts of android module that are unique, like manifest. https://developer.android.com/studio/build/manifest-merge Feel free to link this manifest merge doc, too.